### DISCUSSION TITLE
Welcome to AMBF Discussions!

### DISCUSSION THREAD

**Discussion by adnanmunawar** [WPI-AIM/ambf#151] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shyam-ros-developer** [comment]
Hello, sir. I have learnt some basic animation in blender and I have learnt how to rig wheels and joints. What should I do in blender to make it into a robot that I can launch in AMBF? How do I control it in AMBF? Is it enough if I simply parent appropriate basic meshes? If so, how do I control it in AMBF? 
Please help me, sir.
Sincerely,
Shyam.

**Comment by adnanmunawar** [comment]
Hello Shyam,

AMBF has a Blender [ambf_addon](https://github.com/WPI-AIM/ambf_addon) to load, create and save robots. Please go through the documentation and video tutorials which should help you. Once you have the robot imported inside AMBF, you can use the [Python client](https://github.com/WPI-AIM/ambf/wiki/The-Python-Client) to control it. There are also a few additional examples on the [Wiki](https://github.com/WPI-AIM/ambf/wiki) page on controlling different robots using the Python client.

**Reply by shyam-ros-developer** [reply]
Thank you for your quick response, sir.
I want to build my own model in blender, sir. Can you guide me on how to learn that, sir?
I read the README file, but didn't check out the wiki page, sir. Thank you for that one, sir.

Sincerely,
Shyam.
